lib/screens/about/about.dart

```diff
@@ -11,28 +11,20 @@ import 'package:flutter/material.dart';
 import 'package:redux/redux.dart';
 
 
-class About extends StatefulWidget {
-  @override
-  _AboutState createState() => _AboutState();
-}
-
-class _AboutState extends State<About> {
-
-  List<Contributor> contributors = new List();
+class About extends StatelessWidget {
 
-  Widget contributorsWidget(List<Contributor> contributors) {
+  Widget contributorsWidget() {
     return new StoreConnector<AppState, AppState>(
         converter: (Store<AppState> store) => store.state,
         onInit: (store) {
           store.dispatch(LoadContributorsAction());
         },
         builder: (BuildContext context, AppState state) {
-          if (state.hasContributors == true){
+          if (state.hasContributors){
             return new StoreConnector<AppState, List>(
                 converter: (Store<AppState> store) => store.state.contributors,
                 builder: (BuildContext context, List contributors) {
                   if (contributors != null) {
-                    contributors = contributors;
```

Comment:
So this wasn't needed at all?

Reply:
if only each contributor item need the contributor info, i don't think we need to preserve the list on the parent widget, but correct me if i am wrong!

Reply:
Yes. That makes sense to me as well.